crates/sui-indexer/migrations/2023-03-20-041133_epoch/up.sql

```diff
@@ -1,24 +1,31 @@
 CREATE MATERIALIZED VIEW epoch_network_metrics as
-SELECT MAX(total_successful_transactions * 1000.0 / time_diff)::float8 as tps_30_days
-FROM (
+WITH checkpoints_30d AS (
```

Comment:
how often do we run this query/refresh this view and the recent tps query?

Reply:
this is run every epoch before https://github.com/MystenLabs/sui/pull/12870 and each 100 checkpoints afterwards